### ISSUE TITLE
Business Central Wave 2 issue 'RunOnClient' is not a valid attribute on variables of type 'None'

### ISSUE THREAD

**Issue by rishitiwari1109** [microsoft/AL#5671]
**Describe the bug**
A lot of Base App Objects are creationg problems One example is 'RunOnClient' is not a valid attribute on variables of type 'None'
**To Reproduce**
Steps and to reproduce the behavior:
1. Go to ''

``` AL Code to reproduce the issue ```

**Expected behavior**
There should not be any problem while opening Base Application in Visual Studio
**Screenshots**
![image](https://user-images.githubusercontent.com/60818730/74587187-8985bd00-5015-11ea-974c-dc7cb94e5580.png)

**5. Versions:**

 - AL Language: 4.0.206840

 - Business Central:15.2.0.0

**Comment by atoader** [comment]
RunOnClient is an attribute that applies to .NET types. Have you followed the doc at https://docs.microsoft.com/en-us/dynamics365/business-central/dev-itpro/developer/devenv-get-started-call-dotnet-from-al to configure your environment?

**Comment by atoader** [comment]
I am closing this issue because it appears that it has been resolved. Please open a new issue if you believe this has not been resolved and reference the current issue.

**Comment by anhquandovu** [comment]
Hi, 

I am running into the same issue. Let me describe below:

**1. Task: Technical Upgrade From Version 14 to Version 16**
https://docs.microsoft.com/en-us/dynamics365/business-central/dev-itpro/upgrade/upgrade-technical-upgrade-v14-v16
**- Step 2: Code Conversion from C/AL to AL**
https://docs.microsoft.com/en-us/dynamics365/business-central/dev-itpro/upgrade/devenv-code-conversion
  + I have converted the C/AL TXT files to AL
  + I have created and built an AL project for custom base application
  + In the dotnet.al file for the project: I have deleted all instances of Version = '14.0.0.0'; for Microsoft.Dynamics.Nav assembly declarations.
  For instance:
![image](https://user-images.githubusercontent.com/52545960/89627085-fed69a00-d8a2-11ea-8699-48be316834d9.png)

  + When I built and compiled project, I got hundreds of errors, all of them seemed related to deleted instances of version '14.0.0.0'
![image](https://user-images.githubusercontent.com/52545960/89627652-da2ef200-d8a3-11ea-8693-89613fbc907a.png)

Is there a better way to handle these errors?
I was trying to fix one by one but figured that fixing one would affect to many others and probably would take a lot of time.
